src/ontology/mondo-edit.obo

```diff
@@ -498206,7 +498208,7 @@ xref: OMIM:619661 {source="MONDO:equivalentTo"}
 xref: UMLS:C5562044 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1794254"}
 is_a: MONDO:0030796 {source="OMIM:619661"} ! leukoencephalopathy, hereditary diffuse, with spheroids
 is_a: MONDO:1010132 {source="PMID:31775912"} ! AARS1-related leukoencephalopathy
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/20 {source="OMIM:619661"} ! AARS1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/20 ! AARS1
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8482" xsd:anyURI
```

Comment:
The OMIM source was removed because this is marked as provisional (https://omim.org/entry/619661). However, the gene association was added by user request in this [ticket](https://github.com/monarch-initiative/mondo/issues/8482).

Is there a different source I should add?

Reply:
I added the PMID used for the "AARS1-related leukoebcephalopathy" parent